streams/sql/src/main/java/com/hortonworks/streamline/streams/sql/runtime/CorrelatedEventsAwareValues.java

```diff
@@ -0,0 +1,36 @@
+package com.hortonworks.streamline.streams.sql.runtime;
+
+import com.hortonworks.streamline.streams.StreamlineEvent;
+
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
Can you add some comment on why this is needed?

Reply:
OK will add.